test/integration/controller/core/clusterqueue_controller_test.go

```diff
@@ -758,6 +761,100 @@ var _ = ginkgo.Describe("ClusterQueue controller", ginkgo.Ordered, ginkgo.Contin
 				},
 			}, util.IgnoreConditionTimestampsAndObservedGeneration))
 		})
+
+		ginkgo.It("Should prevent workload admission due to multikueue contraints", func() {
```

Comment:
We want to have also an integration test when the feature gate is not enabled, and we see the reasons are specific for MultiKueue.

Reply:
this is actually the test without the feature gate,
we don't check status there just messages

Reply:
I would say the reason is more important than the message.